src/bin/node.rs

```diff
@@ -153,6 +154,8 @@ async fn main() -> Result<()> {
         })
         .await?;
 
+    handler.build_module::<TcpServer>(ctx.clone()).await?;
```

Comment:
make it optional with a conf ?
(disabled by default IMHO)

Reply:
Also add it to the integration test (disabled by default?)